frontend/packages/vkt/src/components/publicEnrollment/steps/PreviousEnrollment.tsx

```diff
@@ -0,0 +1,41 @@
+import { ChangeEvent } from 'react';
+import { CustomTextField, Text } from 'shared/components';
+
+import { usePublicTranslation } from 'configs/i18n';
+import { useAppDispatch } from 'configs/redux';
+import { PublicEnrollment } from 'interfaces/publicEnrollment';
+import { updatePublicEnrollment } from 'redux/reducers/publicEnrollment';
+
+export const PreviousEnrollment = ({
+  enrollment,
+  editingDisabled,
+}: {
+  enrollment: PublicEnrollment;
+  editingDisabled: boolean;
+}) => {
+  const { t } = usePublicTranslation({
+    keyPrefix: 'vkt.component.publicEnrollment.steps.previousEnrollment',
+  });
+
+  const dispatch = useAppDispatch();
+
+  const handleChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
+    dispatch(
+      updatePublicEnrollment({
+        previousEnrollment: event.target.value,
+      })
+    );
+  };
+
+  return (
+    <div className="public-enrollment__grid__previous-enrollment rows gapped-sm">
+      <Text>{t('description')}</Text>
+      <CustomTextField
```

Comment:
Pikku juttu mutta inputin yläpuolella vois olla ehkä enemmän marginia/tyhjää tilaa. Ainakin safarilla näytti olevan aika kiinni siinä ohjetekstissä inputti.

Reply:
Näyttäis omalla Chromella ja Safarilla olevan saman verran tilaa näiden elementtien välissä. Mut ei oo tosiaan suurta gappia ja tarkoituksella laitoin pienen. Ehkä makuasia ja pitää katsoa Otonkin kanssa miten tän pitäis mennä. Mut voin vaihtaa normaalin gapin tilalle.